docs/user/gcp/iam.md

```diff
@@ -12,7 +10,8 @@ Create a GCP service account using the console or the CLI
 
 ## Step 2: Assign Project Roles to your Service Account
 
-You need to assign the proper roles to the newly created service account so it can be used to create an OpenShift cluster.
+A service account needs to be granted permission for specific APIs in order to access the services used to created a cluster. You can assign the proper roles during or after the creation of a service account. The simplest approach to assigning roles would be to make the service account an Owner of the project, but that service account would then have complete control over the project, which would be a decided risk.
```

Comment:
Is there a point in installation where you can remove the Owner role if you grant it here?

Reply:
https://github.com/openshift/installer/blob/master/docs/user/gcp/postinstall.md